src/container/DeleteTask.js

```diff
@@ -0,0 +1,15 @@
+import { connect } from 'react-redux';
+
+import { deleteTask } from 'src/todoSlice/TodoListSlice';
+import DeleteTask from 'src/components/DeleteTask/DeleteTask';
+
+const mapStateToProps = (state) => ({
+    tasks: state.todoList.tasks,
```

Comment:
similarly we don't need to pass tasks to delete ocmponent, as it only requires the handler function and taskId